### PR TITLE
Changed thresholds.

### PR DESCRIPTION
**Pull request by mehrdadhalali**

### CODE REVIEW

File: dashboard/transform_dashboard.py [hunk 1]
```diff
@@ -98,7 +98,7 @@ def create_moisture_bar(df: pd.DataFrame) -> alt.Chart:
 
     df['at'] = pd.to_datetime(df['at'])
 
-    df["low_moisture"] = df["soil_moisture"] <= 10
+    df["low_moisture"] = df["soil_moisture"] <= 50
```
Comment (YakMan101) [inline dashboard/transform_dashboard.py:101]: Nice, more appropriate thershold.

**Review by YakMan101 — APPROVED** [review]

File: dashboard/transform_dashboard.py (hunk 1 above)
Comment (olliejth) [reply dashboard/transform_dashboard.py:101]: Agree

**Review by olliejth — APPROVED** [review]
I agree with these new thresholds

**Review by Leon-JS02 — APPROVED** [review]
I agree that these are better thresholds.

**Pull request merged**